src/Api/Service/MyParcelApiService.php

```diff
@@ -45,10 +54,35 @@ public function getBaseUrl(): string
     /**
      * @return array
      */
-    protected function getHeaders(): array
+    public function getHeaders(): array
     {
         return [
             'authorization' => sprintf('appelboom %s', base64_encode($this->apiKey)),
+            'User-Agent'    => $this->getUserAgentHeader(),
         ];
     }
+
+    /**
+     * @return string
+     */
+    protected function getUserAgentHeader(): string
+    {
+        $userAgentStrings = [];
+        $userAgents       = array_merge(
+            [$this->userAgent],
```

Comment:
Maak hier een array van zodat dit simpeler kan, of zet dit alvast in `$userAgentStrings` en niet deze array